webapp/root.jsx

```diff
@@ -228,6 +234,42 @@ function renderRootComponent() {
                             center: null
                         }}
                     />
+                    <Route path=':team/integrations'>
```

Comment:
I think this needs to be another level deep. Especially when we remove `/:team/`. 
Something like `/settings/integrations`